site/content/how-to/traffic-management/routing-traffic-to-your-app.md

```diff
@@ -0,0 +1,371 @@
+---
+title: "Routing Traffic to Your Application"
```

Comment:
Would it make sense to create a new section for Security-related guides and put this guide there? That section will be separate from traffic management.

Like this one https://docs.nginx.com/nginx/

Reply:
See my other comment about the scope of this PR. Let's move the conversation about IA changes to slack and a separate PR. cc @ADubhlaoich